### PR TITLE
data-quality-report:  Improvements

### PR DESCRIPTION
**Pull request by ghost**
Hide status fields (they don't make sense now) and add public download

### CODE REVIEW

**Comment by ghost** [comment]
Build in background and store in cache (instead of build on demand). Building once per day is fine. We are going to add more stats so build times will only get longer - currently 10s on my laptop.

**Comment by ghost** [comment]
Also noting there is an inefficiency here (we look up how many public projects there are repeatedly when building) but as it builds in the background I'm not bothered at this stage.

**Review by Bjwebb — APPROVED** [review]

File: indigo/files.py [hunk 1]
```diff
@@ -309,3 +312,42 @@ def _put_file_in_zip_file(zipfiles, file_name_in_storage, file_name_in_zip):
...
+
+    # Data
+    data = {"single_fields": []}
+
+    fields = [
+        i
+        for i in settings.JSONDATAFERRET_TYPE_INFORMATION["project"]["fields"]
+        if i.get("type") != "list"
+        # Because we only generate stats on public data anyway, running stats on the status field makes no sense.
+        and i.get("key") != "/status"
```
Comment (Bjwebb) [inline indigo/files.py:327]: This is made redundant by the line below.

File: indigo/views.py [hunk 2]
```diff
@@ -1067,6 +1077,8 @@ def admin_all_projects_data_quality_report(request):
                 if i.get("type") != "list"
                 # Because we only generate stats on public data anyway, running stats on the status field makes no sense.
                 and i.get("key") != "/status"
```
Comment (Bjwebb) [inline indigo/views.py:1079]: This is made redundant by the line below.

File: indigo/files.py (hunk 1 above)
Comment (ghost) [reply indigo/files.py:327]: I realised that technically but left for readability, tho I probably should have just tried to deal with that with comments

**Pull request merged**